src/EditorFeatures/CSharpTest/project.json

```diff
@@ -4,6 +4,10 @@
             "version": "14.3.25407",
             "suppressParent": "all"
         },
+        "RoslynDependencies.Microsoft.VisualStudio.Text.Internal": {
+           "version": "14.3.25407",
+           "suppressParent": "all"
+        },
```

Comment:
IIRC the InteractiveWindow uses some types from this, so it doesn't surprise me.